x-pack/plugins/lens/public/indexpattern_datasource/utils.ts

```diff
@@ -53,9 +54,16 @@ export function isColumnInvalid(
   if (!column) return;
 
   const operationDefinition = column.operationType && operationDefinitionMap[column.operationType];
-  return !!(
-    operationDefinition.getErrorMessage &&
-    operationDefinition.getErrorMessage(layer, columnId, indexPattern)
+  // check also references for errors
```

Comment:
Fine for now, but note: Once we introduce formula, we have to extend this to follow references through multiple columns, not just a single "hop"

Reply:
I've refactor that to be in a distinct function, so we could easily change it later.